pkg/operator/ceph/controller/spec_test.go

```diff
@@ -382,63 +382,63 @@ func TestLogCollectorContainer(t *testing.T) {
 	t.Run("Periodicity 1d and no MaxlogSize", func(t *testing.T) {
 		c := cephv1.ClusterSpec{LogCollector: cephv1.LogCollectorSpec{Enabled: true, Periodicity: "1d"}}
 		got := LogCollectorContainer(daemonId, ns, c)
-		want := fmt.Sprintf(cronLogRotate, daemonId, "daily", "0")
+		want := fmt.Sprintf(cronLogRotate, daemonId, "daily", "0", "7")
```

Comment:
It'd be nice to add a unit test for rbd-mirror `daemonId` and `"28"` here.

Reply:
added